Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1043,6 +1046,24 @@ def get_full_report_command(
         ]
     return command_results, is_command_finished
 
+def validate_sandbox_report(report_resources: list[dict[str, Any]]) -> str:
```

Comment:
`validate` method returning a string with the error is a bit unconventional.

Instead, please create a custom exception type (above) and raise it here (I'd include the word `Warning` in its name), catch wherever this code is used and act (`return_warning`) accordingly.